### DISCUSSION TITLE
How to ignore unknown elements?

### DISCUSSION THREAD

**Discussion by Kryomancer** [pdvrieze/xmlutil#125] [Q&A] [answered]
I have a large, sprawling XML file but I'm only interested in certain tags, is there any way to avoid having to write a class or property for every single one and simply ignore anything that doesn't have a matching field on the Kotlin side of things? I do not need this to support serializing back into the original in any way,

**Comment by pdvrieze** [comment, marked as answer]
Yes there is:

When you create the format implementation you can configure it with a policy:

```kotlin
XML {
    defaultPolicy {
        pedantic = false
        ignoreUnknownChildren() // or ignoreNamespaces() if only tags in other namespaces are to be ignored
    }
}
```
// alternatively you can set `unknownChildHandler` explicitly to any value you want.
